services/cli/src/utils/rebalancing_simulator.rs

```diff
@@ -1035,6 +1109,34 @@ impl RebalancingSimulator {
         epoch_map
     }
 
+    /// This returns the hashmap of manual withdraws and deposits of SOL epochwise
```

Comment:
I might modify this comment to be more clear. From reviewing the function I would say it maps epochs to their SOL deposits/withdrawals and the active_stake at that epoch.